docs/docs/Get-Started/get-started-installation.md

```diff
@@ -6,166 +6,228 @@ slug: /get-started-installation
-* As a [cloud-hosted service](#datastax-langflow) with DataStax Langflow
+* **Recommended option**: Download and install the [standalone desktop application](#install-and-run-langflow-desktop) for the easiest setup experience.
 
-## Install and run Langflow OSS
+* **Docker**: Pull and run the [Docker image](#install-and-run-langflow-docker) to start a Langflow container.
 
-Before you install and run Langflow OSS, be sure you have the following items.
+* **For developers**: Install the [Langflow OSS Python package](#install-and-run-langflow-oss).
 
-- [Python 3.10 to 3.13](https://www.python.org/downloads/release/python-3100/)
-- [uv](https://docs.astral.sh/uv/getting-started/installation/) or [pip](https://pypi.org/project/pip/)
-- A virtual environment created with [uv](https://docs.astral.sh/uv/pip/environments) or [venv](https://docs.python.org/3/library/venv.html)
-- A dual-core CPU and at least 2 GB of RAM. More intensive use requires a multi-core CPU and at least 4 GB of RAM.
+## Install and run Langflow Desktop
 
-Install and run Langflow OSS with [uv (recommended)](https://docs.astral.sh/uv/getting-started/installation/) or [pip](https://pypi.org/project/pip/).
+**Langflow Desktop** is a desktop version of Langflow that includes all the features of open source Langflow, with an additional [version management](#manage-your-langflow-version-in-langflow-desktop) feature for managing your Langflow version.
 
-1. To install Langflow, use one of the following commands:
+<Tabs groupId="os">
+  <TabItem value="macOS" label="macOS">
 
-<Tabs groupId="package-manager">
-<TabItem value="uv" label="uv" default>
+  1. Navigate to [Langflow Desktop](https://www.langflow.org/desktop).
+  2. Enter your **Name**, **Email address**, and **Company**, and then click **Download**.
+  3. Open the **Finder**, and then navigate to **Downloads**.
+  4. Double-click the downloaded `.dmg` file.
+  5. To install Langflow Desktop, drag and drop the application icon to the **Applications** folder.
+  6. When the installation completes, open the Langflow application.
 
-```bash
-uv pip install langflow
-```
+  </TabItem>
+  <TabItem value="Windows" label="Windows">
 
-</TabItem>
-<TabItem value="pip" label="pip">
+  1. Navigate to [Langflow Desktop](https://www.langflow.org/desktop).
+  2. Enter your **Name**, **Email address**, and **Company**, and then click **Download**.
+  3. Open the **File Explorer**, and then navigate to **Downloads**.
+  4. Double-click the downloaded `.msi` file.
+  5. To install Langflow Desktop, follow the instructions to install the executable file.
+  6. When the installation completes, open the Langflow application.
 
-```bash
-pip install langflow
-```
+  </TabItem>
 
-</TabItem>
 </Tabs>
 
-2. To run Langflow, use one of the following commands:
+The application checks [uv](https://docs.astral.sh/uv/concepts/tools/), your local environment, and the Langflow version, and then starts.
```

Comment:
You mention uv here but don't state that it's required to install LF Desktop. Do they need to have uv installed or will the LF Desktop command install it as a dependency?

Reply:
LF Desktop installs it